browser/brave_browser_process_impl.cc

```diff
@@ -284,6 +287,16 @@ BraveBrowserProcessImpl::https_upgrade_exceptions_service() {
   return https_upgrade_exceptions_service_.get();
 }
 
+webcompat_exceptions::WebcompatExceptionsService*
+BraveBrowserProcessImpl::webcompat_exceptions_service() {
+  if (!webcompat_exceptions_service_) {
```

Comment:
i think we need a `base::Feature` for this

Reply:
Added.